samples/caf/boards/nrf5340dk_nrf5340_cpuapp.overlay

```diff
@@ -39,30 +39,92 @@
 
 &pwm0 {
 	status = "okay";
-	ch0-pin = <28>;
-	ch0-inverted;
+	pinctrl-0 = <&pwm0_default>;
+	pinctrl-1 = <&pwm0_sleep>;
+	pinctrl-names = "default", "sleep";
 };
 
 &pwm1 {
 	status = "okay";
-	ch0-pin = <29>;
-	ch0-inverted;
+	pinctrl-0 = <&pwm1_default>;
+	pinctrl-1 = <&pwm1_sleep>;
+	pinctrl-names = "default", "sleep";
 };
 
 &pwm2 {
 	status = "okay";
-	ch0-pin = <30>;
-	ch0-inverted;
+	pinctrl-0 = <&pwm2_default>;
+	pinctrl-1 = <&pwm2_sleep>;
+	pinctrl-names = "default", "sleep";
 };
 
 &pwm3 {
 	status = "okay";
-	ch0-pin = <31>;
-	ch0-inverted;
+	pinctrl-0 = <&pwm3_default>;
+	pinctrl-1 = <&pwm3_sleep>;
+	pinctrl-names = "default", "sleep";
 };
 
 &pwm_led0 {
 	status = "okay";
 	pwms = <&pwm0 28>;
 	label = "LED 1";
 };
+
+&pinctrl {
+	pwm0_default: pwm0_default {
+		group1 {
+			psels = <NRF_PSEL(PWM_OUT0, 0, 28)>;
+			nordic,invert;
+		};
+	};
+
+	pwm0_sleep: pwm0_sleep {
+		group1 {
+			psels = <NRF_PSEL(PWM_OUT0, 0, 28)>;
+			low-power-enable;
+		};
+	};
+
+	pwm1_default: pwm1_default {
+		group1 {
+			psels = <NRF_PSEL(PWM_OUT1, 0, 29)>;
```

Comment:
It should be `PWM_OUT0`, for all PWM instances, as originally only `ch0-*` properties were used.

Reply:
And here.